script.module.inputstreamhelper/default.py

```diff
@@ -4,7 +4,16 @@
 from __future__ import absolute_import, division, unicode_literals
 import os
 import sys
-sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'lib'))  # noqa: E402
-from inputstreamhelper.api import run
+import xbmc
+import xbmcaddon
+
+
+def to_unicode(text, encoding='utf-8'):
+    ''' Force text to unicode '''
+    return text.decode(encoding) if isinstance(text, bytes) else text
+
+
+sys.path.append(os.path.join(to_unicode(xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path'))), 'lib'))
```

Comment:
Why not just creating a `python.module` extension point instead of "injecting" the path? Something like youtube does: https://github.com/jdf76/plugin.video.youtube/blob/master/addon.xml#L13

I also don't see why you need to append to `sys.path` within the same addon scope.

Reply:
I use a `xbmc.python.script` extension point pointing to `default.py` in the add-on root directory and a `xbmc.python.module` pointing to the `lib` directory and it turned out I can't access the `lib` directory from `default.py`.